gp2gp-translator/src/integrationTest/java/uk/nhs/adaptors/pss/translator/amqp/ServiceFailureIT.java

```diff
@@ -73,11 +78,14 @@ public class ServiceFailureIT extends BaseEhrHandler {
 
     @Mock
     private HttpHeaders httpHeaders;
+
     @SpyBean
     private MhsClientService mhsClientService;
     @SpyBean
     private SendContinueRequestHandler sendContinueRequestHandler;
     @SpyBean
+    private COPCMessageHandler copcMessageHandler;
```

Comment:
So I added this when testing on code with me, though it is not in use. Would it be worth removing this, following `YAGNI`?